ecs/ec2.go

```diff
@@ -41,14 +53,20 @@ func (b *ecsAPIService) createCapacityProvider(ctx context.Context, project *typ
 		return nil
 	}
 
-	ami, err := b.SDK.GetParameter(ctx, "/aws/service/ecs/optimized-ami/amazon-linux-2/gpu/recommended")
-	if err != nil {
-		return err
+	if ami != "" {
```

Comment:
At first glance it looks like we discard the user defined `ami` value and get the recommended one, or am I misunderstanding?

Reply:
indeed, should be `==`. Good catch